From: Jan Engelhardt <ej@inai.de>
Date: 2025-07-17 10:15:40.088500829 +0200
References: https://github.com/uxlfoundation/oneTBB/issues/1783

-fcf-prot: not supported on this target
[at least i586, armv7hl, ppc64le, s390x, probably also riscv64]
---
 cmake/compilers/GNU.cmake |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: oneTBB-2022.2.0/cmake/compilers/GNU.cmake
===================================================================
--- oneTBB-2022.2.0.orig/cmake/compilers/GNU.cmake
+++ oneTBB-2022.2.0/cmake/compilers/GNU.cmake
@@ -107,7 +107,7 @@ endif ()
 set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -fno-strict-overflow -fno-delete-null-pointer-checks -fwrapv)
 set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -Wformat -Wformat-security -Werror=format-security
     -fstack-protector-strong )
-if (NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64" AND NOT EMSCRIPTEN)
+if (FALSE)
     set(TBB_LIB_COMPILE_FLAGS ${TBB_LIB_COMPILE_FLAGS} $<$<NOT:$<VERSION_LESS:${CMAKE_CXX_COMPILER_VERSION},8.0>>:-fcf-protection=full>)
 endif ()
 set(TBB_LIB_COMPILE_FLAGS ${TBB_LIB_COMPILE_FLAGS} $<$<NOT:$<VERSION_LESS:${CMAKE_CXX_COMPILER_VERSION},8.0>>:-fstack-clash-protection>)
